README.md

```diff
@@ -4,26 +4,25 @@
 |:--:|:--:|
 |[![Master Build Status](https://travis-ci.org/OmniSharp/omnisharp-vscode.svg?branch=master)](https://travis-ci.org/OmniSharp/omnisharp-vscode)|[![Dev Build Status](https://travis-ci.org/OmniSharp/omnisharp-vscode.svg?branch=dev)](https://travis-ci.org/OmniSharp/omnisharp-vscode)|
 
-This is a preview of C# support in Visual Studio code. It is designed to work well with [.NET Core 1.0](http://dotnet.github.io/). 
-These new projects are based on project.json and the [.NET CLI](https://dotnet.github.io/getting-started/).
+Welcome to the C# extension for Visual Studio Code! This preview provides the following features inside VS Code:
 
-### Support for .csproj and Mono on OS X and Linux
+* Lightweight development tools for [.NET Core](https://dotnet.github.io).
+* Great C# editing support, including Syntax Highlighting, IntelliSense, Go to Definition, Find All References, etc.
+* Debugging support for .NET Core projects targeting CoreCLR.
```

Comment:
True. I just wanted to call out that it's a specifically a CoreCLR debugger. After all, the C# extension can load .NET framework projects on Windows.